include/nanobind/ndarray.h

```diff
@@ -293,9 +293,12 @@ template <typename... Args> class ndarray {
     }
 
     dlpack::dtype dtype() const { return m_dltensor.dtype; }
+    size_t size() const { return Info::size; }
```

Comment:
Isn't `size()` doing the same as `ndim()` here? I would have expected `size()` to return the total number of tensor coefficients.